src/main/java/org/sagebionetworks/bridge/models/permissions/AccessLevel.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.bridge.models.permissions;
+
+public enum AccessLevel {
+    
+    LIST,
+    READ,
+    EDIT,
+    DELETE,
+    ADMIN
```

Comment:
We should document, as in much detail as we can, what each of these permissions means, and we'll need to copy it all over to the public documentation website.

Reply:
I added some light descriptions here. Definitely going to need a lot more. The combination of accessLevel and entityType is going to need a lot of attention.